docs/source/features/cli.rst

```diff
@@ -194,3 +112,56 @@ Manage the AzureML Compute resources.
     :func: get_cli_parser
     :prog: olive
     :path: manage-aml-compute
+
+Input Model
+===========
+
+There are more than one way to supply input model to the Olive commands.
+
+1. HuggingFace model can be directly used as an input model. For example ``-m microsoft/Phi-3-mini-4k-instruct``.
+
+2. A model produced by a Olive command can be directly used as an input model. You can specify the model file path using the ``-m <output_model>`` option, where ``<output_model>`` is the output folder defined by ``-o <output_model>`` in the previous Olive command.
+
+3. Olive commands also accept a local PyTorch model as an input model. You can specify the model file path using the ``-m model.pt`` option, and the associated model script using the ``--model_script script.py`` option. For example, ``olive capture-onnx-graph -m model.pt --model_script script.py``.
+
+4. A model from AzureML registry can be directly used as an input model. For example ``-m azureml://registries/<registry_name>/models/<model_name>/versions/<version>``.
+
+5. An ONNX model available locally can also be used as an input for the Olive commands that accept ONNX model as an input.
```

Comment:
Local HF model can be accepted also (not trying this, i can try this later).

Reply:
ok